Telerik.Examples.Mvc/Telerik.Examples.Mvc/Views/Shared/_Layout.cshtml

```diff
@@ -5,9 +5,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@ViewData["Title"] - Telerik.Examples.Mvc</title>
     @{
-        var kendoVersion = "2023.3.1114";
+        var kendoVersion = "2025.2.520";
+        var themeVersion = "11.0.2";
     }
-    <link rel="stylesheet" href="https://kendo.cdn.telerik.com/themes/7.0.2/default/default-ocean-blue.css">
+    <link href="https://kendo.cdn.telerik.com/themes/@themeVersion/default/default-ocean-blue.css" rel="stylesheet" type="text/css" />

```

Comment:
Update the reference to the JSZip library (<script src="https://unpkg.com/jszip/dist/jszip.min.js"></script>)